### ISSUE TITLE
Twig runtime error editing submissions from front end

### ISSUE THREAD

**Issue by d-karstens** [verbb/formie#349]
**Description**
When attempting to edit submissions from the front end using the guide in the docs we've been running into the following twig runtime error when the form has a tag field

<img width="1250" alt="Screen Shot 2021-02-16 at 11 08 07 am" src="https://user-images.githubusercontent.com/21274925/108013432-9d666380-705b-11eb-937b-0d494df153e5.png">

**Steps to reproduce**
1. Create form with a tag field
2. Set up a page to edit submissions from front end using https://verbb.io/craft-plugins/formie/docs/template-guides/editing-submissions

### Form settings
Please pick from the options below.

- Multi-page form: No
- Submission Method: Ajax
- Client-side Validation: No
- Custom Form Templates: No

**Additional info**
- Plugin version: 1.3.17
- Craft version: 3.6.5.1
- Multi-site: No

**Additional context**

**Comment by engram-design** [comment]
Should be fixed for the next release. To get the fix early, change your `verbb/formie` requirement in `composer.json` to:

```
"require": {
  "verbb/formie": "dev-craft-3 as 1.3.17",
  "...": "..."
}
```

Then run `composer update`.

**Comment by d-karstens** [comment]
Thanks, very much appreciated!

**Comment by engram-design** [comment]
Fixed in [1.3.18](https://github.com/verbb/formie/releases/tag/1.3.18)
